tests/unit/components/pages/donation_form/DonationFormReceiptAlternateOrder.spec.ts

```diff
@@ -0,0 +1,469 @@
+		await jest.runAllTimersAsync();
+
+		await wrapper.find( '#donation-form' ).trigger( 'submit' );
+		await nextTick();
+		await nextTick();
+
+		expect( errorSummaryItemIsFunctional( wrapper, 'paymentType-0', 'payment-form-type-scroll-target' ) ).toBeTruthy();
+
+		// Make the IBAN field appear
+		await wrapper.find( 'input[name="paymentType"][value="BEZ"]' ).trigger( 'change' );
+		await wrapper.find( '#donation-form' ).trigger( 'submit' );
+		await nextTick();
+		await nextTick();
+
+		expect( wrapper.find( '.error-summary' ).exists() ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'amount-500', 'payment-form-amount-scroll-target' ) ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'account-number', 'account-number-scroll-target' ) ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'salutation-0', 'salutation-scroll-target' ) ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'first-name', 'first-name-scroll-target' ) ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'last-name', 'last-name-scroll-target' ) ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'email', 'email-scroll-target' ) ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'street', 'street-scroll-target' ) ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'post-code', 'post-code-scroll-target' ) ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'city', 'city-scroll-target' ) ).toBeTruthy();
+		expect( errorSummaryItemIsFunctional( wrapper, 'country', 'country-scroll-target' ) ).toBeTruthy();
+
+		await wrapper.find( 'input[name="amount"][value="500"]' ).trigger( 'change' );
+		await wrapper.find( 'input[name="salutation"][value="Mr"]' ).trigger( 'change' );
+
+		await wrapper.find( '#account-number' ).setValue( IBAN );
+		await wrapper.find( '#account-number' ).trigger( 'blur' );
+
+		await wrapper.find( '#first-name' ).setValue( 'first-name' );
+		await wrapper.find( '#first-name' ).trigger( 'blur' );
+
+		await wrapper.find( '#last-name' ).setValue( 'last-name' );
+		await wrapper.find( '#last-name' ).trigger( 'blur' );
+
+		await wrapper.find( '#street' ).setValue( 'street' );
+		await wrapper.find( '#street' ).trigger( 'blur' );
+
+		await wrapper.find( '#post-code' ).setValue( '12345' );
+		await wrapper.find( '#post-code' ).trigger( 'blur' );
+
+		await wrapper.find( '#city' ).setValue( 'city' );
+		await wrapper.find( '#city' ).trigger( 'blur' );
+
+		await wrapper.find( '#country' ).setValue( countries[ 0 ].countryFullName );
+		await wrapper.find( '#country' ).trigger( 'blur' );
+
+		await wrapper.find( '#email' ).setValue( 'joe@dolan.com' );
+		await wrapper.find( '#email' ).trigger( 'blur' );
+
+		await jest.runAllTimersAsync();
+
+		expect( wrapper.find( '.error-summary' ).exists() ).toBeFalsy();
+	} );
+
+	it( 'handles the error summary when company address type was selected selected before submitting', async () => {
```

Comment:
```suggestion
	it( 'handles the error summary when company address type was selected before submitting', async () => {
```
there might be more occurences..